Cake.XComponent/Utils/PathFinder.cs

```diff
@@ -8,8 +8,12 @@ namespace Cake.XComponent.Utils
 {
     internal class PathFinder
     {
-        internal const string XcStudioExe = "XCStudio.exe";
-        internal const string XcBuildExe = "xcbuild.exe";
+        private const string XcStudioProgram = "XCStudio{0}.exe";
+        private const string XcSpyProgram = "xcspy{0}.exe";
```

Comment:
That's an inconsistency to fix (eventually) on the executable... sometimes it's XC, sometimes xc